packages/frontend/src/lib/ErrorMessage.svelte

```diff
@@ -5,17 +5,24 @@ import Tooltip from './Tooltip.svelte';
 
 export let error: string;
 export let icon = false;
+export let tooltipPosition: 'top' | 'topLeft' | 'bottomLeft' = 'top';
```

Comment:
Why do you change the ErrorMessage here ?

Reply:
TBH i didn't think it was in the UI package, i'll reset it